SPECS/mariner-repos/mariner-extras.repo

```diff
@@ -0,0 +1,10 @@
+[mariner-extras]
+name=Mariner Extras $basearch
+baseurl=https://packages.microsoft.com/cbl-mariner/$releasever/mariner-extras/base/$basearch/rpms
```

Comment:
Do we already have a path set for the extras? I'm curious, if it shouldn't be under `.../$releasever/prod/extras/$basearch/...`, so that it's right next to the `base` and `update` ones.